pkg/execution/state/redis_state/lua/queue/backlogRefill.lua

```diff
@@ -0,0 +1,78 @@
+local keyBacklogSet                      = KEYS[1]
+local keyShadowPartitionSet              = KEYS[2]
+local keyGlobalShadowPartitionSet        = KEYS[3]
+local keyGlobalAccountShadowPartitionSet = KEYS[4]
+local keyAccountShadowPartitionSet       = KEYS[5]
+local keyReadySet                        = KEYS[6]
+
+local backlogID   = ARGV[1]
+local partitionID = ARGV[2]
+local accountID   = ARGV[3]
+
+-- $include(update_pointer_score.lua)
+-- $include(ends_with.lua)
+-- $include(update_account_queues.lua)
+
+local capacity = 100
+
+-- TODO Enforce constraints and adjust capacity
+
+-- TODO Move item(s) out of backlog and into partition
+
+local items = redis.call("ZRANGE", keyBacklogSet, "-inf", "+inf", "BYSCORE", "LIMIT", 0, capacity, "WITHSCORES")
+
+-- Reverse the items to be added to the ready set
+local args = {}
+local remArgs = {}
+for i = 1, #items, 2 do
```

Comment:
why increment by 2 here?

Reply:
Every second item is the score

Reply:
So we need to skip by two every time

Reply:
oh I see. better to put that into comments.